### DISCUSSION TITLE
求一个把专栏从图文中分离出来的功能

### DISCUSSION THREAD

**Discussion by ChenJW431** [the1812/Bilibili-Evolved#4598] [功能建议 / Ideas]
现在的图文真是负优化，页面极为混乱，而且找不到up主写的专栏了。求作者做一个把专栏从图文分离，筛选的功能。

**Comment by Geeyun-JY3** [comment]
破站这改的什么玩意，合并专栏文章和带图动态就算了还是瀑布流，产品经理没事吧😅。

**Comment by the1812** [comment]
相关背景: https://github.com/the1812/Bilibili-Evolved/discussions/4596

**Comment by Geeyun-JY3** [comment]
昨天发现破站终于在图文里给了分类 tab 可以只看专栏了，新旧个人空间都有，虽然还是无限滚动瀑布流，还是这种布局。
![image](https://github.com/user-attachments/assets/d0bcdfec-268e-4568-97b2-9e1322487e0c)

笑点解析：这个图文里新加的专栏 tab 在旧版个人空间里有 bug，UP 主无创建任何文集时专栏文章上面会显示个「空间主人还没有投过专栏哦~~」的占位图。
![image](https://github.com/user-attachments/assets/53bd7819-b4bc-42d7-b08b-c047428fb7f3)
